core/rawdb/accessors_chain.go

```diff
@@ -623,7 +623,7 @@ type storedReceiptRLP struct {
 	PostStateOrStatus []byte
 	CumulativeGasUsed uint64
 	Logs              []*types.LogForStorage
-	L1Fee             *big.Int
+	L1DataFee         *big.Int
```

Comment:
```suggestion
	L1Fee         *big.Int
```

This is not used here, but still better to keep it consistent with `core/types/receipt.go`.

Reply:
fixed in https://github.com/scroll-tech/go-ethereum/pull/351/commits/507422f7750cf571a12cae00e2cfc0ad1bffe0c3